### PR TITLE
feat: added tests for bitConvertor

### PR DESCRIPTION
**Pull request by pulkit-30**
Added Tests for bit converter 
- [x] decimal to binary
- [x] decimal to octal
- [x] decimal to hexadecimalmal
- [x] decimal to BCD

### CODE REVIEW

File: simulator/spec/bitConvertor.spec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        expect(() => bitConverterDialog()).not.toThrow();
+    });
+
+    test('function setupBitConvertor working', () => {
+        expect(() => setupBitConvertor()).not.toThrow();
+    });
+
+    test('function setBaseValues working', () => {
+        const randomBaseValue = Math.floor(Math.random() * 100);
+        console.log('Testing for Base Value --> ', randomBaseValue);
```
[the remark this answers is not in the document]
Comment (vedant-jain03) [reply simulator/spec/bitConvertor.spec.js:25]: @pulkit-30 is this console expected?

[the remark this answers is not in the document]
Comment (vedant-jain03) [reply simulator/spec/bitConvertor.spec.js:25]: @pulkit-30 Is this console expected?

**Comment by gr455** [comment]
@pulkit-30 Can you remove the code redundancy from ux.js as we talked on Slack?

File: simulator/spec/bitConvertor.spec.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (pulkit-30) [reply simulator/spec/bitConvertor.spec.js:25]: Yep!, to know for what value(random number) setBaseValue function is being tested.

**Comment by pulkit-30** [comment]
> Can you remove the code redundancy from ux.js as we talked on Slack?

Sure 👍🏽

**Comment by pulkit-30** [comment]
@gr455

**Review by gr455 — APPROVED** [review]
LGTM :tada:

**Pull request merged**